renderapi/transform/leaf/thin_plate_spline.py

```diff
@@ -2,6 +2,8 @@
 from renderapi.errors import RenderError, EstimationError
 from renderapi.utils import encodeBase64, decodeBase64
 from .transform import Transform
+import scipy.spatial
+import scipy.special
```

Comment:
duplicate line

Reply:
it is not duplicate, but I ended up not using scipy.special ... would have thought flake8 would have told me that... will remove